assets/prometheus-operator/deployment.yaml

```diff
@@ -34,6 +34,8 @@ spec:
       - args:
         - --kubelet-service=kube-system/kubelet
         - --prometheus-config-reloader=quay.io/prometheus-operator/prometheus-config-reloader:v0.76.2
+        - --kubelet-endpoints=true
+        - --kubelet-endpointslice=false
```

Comment:
ah we'd need to tweak the jsonnet a bit to remove the flag since it's not yet supported... Or try to merge the prometheus-operator bump before this one.

Reply:
Aah that newly added in operator version right. We can try merging operator PR